api/src/config.ts

```diff
@@ -37,16 +37,20 @@ export const MAX_CHALLENGE_REQUESTS = 100; // maximum number of allowed challeng
 // Secret for Canny SSO
 export const CANNY_KEY = getEnv('CANNY_KEY');
 
+// TODO make sure the frontend port is synchronized with this;
+// we should use the same script to start both frontend and backend
+const port = process.env.FRONT_END_PORT || 3000;
```

Comment:
FYI, I've added a `PORT` env to frontend and `FRONT_END_PORT` to api.

Reply:
great